### PR TITLE
Correcting admin role language

### PR DESCRIPTION
**Pull request by DanPandre**
Global admin is a role not a security group in Azure. The additional note referred to global admin role, but actually means the Device Administrator role (global admin role works fine regardless of role changes after OOBE).

### CODE REVIEW

**Comment by illfated** [comment]
Interesting update, looks very useful. May I request a cosmetic MD Note spacing change, seeing that you are already editing that section?
MarkDown quote markers should ideally have 1 single space between them and any following text (as long as the line is not empty).

My suggestion pertains to your PR lines 342, 343 and 345. If you prefer, I can add suggestions in the source code view.

**Comment by DanPandre** [comment]
Hi Trond,
I’m afraid I don’t know what you mean, I’m new to both MarkDown and GitHub. I just know the Surface Hub product.

Please do add any spacing changes you feel are needed, I’m just trying to make the content more accurate.

Thanks,

Dan Pandre
Sr. Escalation Engineer
Windows<https://docs.microsoft.com/en-us/windows/uwp/get-started/universal-application-platform-guide>.Team<https://support.microsoft.com/hub/4343507> & Holographic<https://support.microsoft.com/hub/4338666> Devices
Customer Service & Support<https://support.microsoft.com/en-us/supportforbusiness/productselection> (Americas)
Hours: 8 AM – 5 PM EST
Phone: (781) 398-4736<tel:17813984736>
Global CSS Phone Numbers<https://support.microsoft.com/help/4051701>
5 Wayside Road, Burlington MA
Email: dpandre@microsoft.com<mailto:dpandre@microsoft.com>
Manager: Peter Dempsey<mailto:peterdem@microsoft.com>
[cid:image002.jpg@01D15E94.64301AE0]

File: devices/surface-hub/first-run-program-surface-hub.md [hunk 1]
```diff
@@ -337,10 +337,10 @@ This is what happens when you choose an option.
...
 
-    Clicking this option allows you to join the device to Azure AD. Once you click **Next**, the device will restart to apply some settings, and then you’ll be taken to the [Use Microsoft Azure Active Directory](#use-microsoft-azure) page and asked to enter credentials that can allow you to join Azure AD. Members of the Azure Global Admins security group from the joined organization will be able to use the Settings app. The specific people that will be allowed depends on your Azure AD subscription and how you’ve configured the settings for your Azure AD organization.
+    Clicking this option allows you to join the device to Azure AD. Once you click **Next**, the device will restart to apply some settings, and then you’ll be taken to the [Use Microsoft Azure Active Directory](#use-microsoft-azure) page and asked to enter credentials that can allow you to join Azure AD. Members of the Azure Global Admins role from the joined organization will be able to use the Settings app. The specific people that will be allowed depends on your Azure AD subscription and how you’ve configured the settings for your Azure AD organization.
     
     >[!IMPORTANT]
```
Comment (illfated) [inline devices/surface-hub/first-run-program-surface-hub.md]: (cosmetic/compatibility spacing)
```suggestion
    > [!IMPORTANT]
```

File: devices/surface-hub/first-run-program-surface-hub.md [hunk 2]
```diff
@@ -337,10 +337,10 @@ This is what happens when you choose an option.
...
-    >Administrators added to the Azure Global Admins group after you join the device to Azure AD will be unable to use the Settings app.
+    >Administrators added to the Azure Device Administrators role after you join the device to Azure AD will be unable to use the Settings app.
```
Comment (illfated) [inline devices/surface-hub/first-run-program-surface-hub.md]: (cosmetic/compatibility spacing)
```suggestion
    > Administrators added to the Azure Device Administrators role after you join the device to Azure AD will be unable to use the Settings app.
```

File: devices/surface-hub/first-run-program-surface-hub.md [hunk 3]
```diff
@@ -337,10 +337,10 @@ This is what happens when you choose an option.
...
     >
     >If you join Surface Hub to Azure AD during first-run setup, single sign-on (SSO) for Office apps will not work properly. Users will have to sign in to each Office app individually.
```
Comment (illfated) [inline devices/surface-hub/first-run-program-surface-hub.md]: (cosmetic/compatibility spacing)
```suggestion
    > If you join Surface Hub to Azure AD during first-run setup, single sign-on (SSO) for Office apps will not work properly. Users will have to sign in to each Office app individually.
```

**Comment by illfated** [comment]
Very well, thank you for your kind and straight forward answer. The changes are not imperative, but if you would be so kind as to implement them by committing the changes, this section will be up to current MarkDown standards. Thank you in advance.

**Pull request merged**